vcpkg.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "merry-mem",
```

Comment:
For the sake of consistency, should we use `snake_case` instead of `kebab-case` here?

Reply:
Unfortunately, vcpkg [requires that names use kebab case](https://learn.microsoft.com/en-us/vcpkg/reference/vcpkg-json#name) lol.

Reply:
That's so annoying. Ugh, Microsoft.